app/__init__.py

```diff
@@ -6,3 +6,5 @@
 BAD_RESPONSE_MSG = {'text': 'There was an error, please enter your access code and try again.', "clickable": True, "level": "ERROR", "type": "SYSTEM_RESPONSE_ERROR"}  # NOQA
 INVALID_CODE_MSG = {'text': 'Please re-enter your access code and try again.', "clickable": True, "level": "ERROR", "type": "INVALID_CODE"}  # NOQA
 NOT_AUTHORIZED_MSG = {'text': 'There was a problem connecting to this study. Please try again later.', "level": "ERROR", "type": "SYSTEM_AUTH_ERROR"}  # NOQA
+
+MAINTENANCE_MSG = {"text": "This site will be temporarily unavailable for maintenance <b>{custom_message}</b>.\nWe apologise for any inconvenience this may cause.", "level": "INFO", "type": "PLANNED_MAINTENANCE"}  # NOQA
```

Comment:
@jcox-dev Need to use ```<strong>``` instead of ```<b>``` as it's to provide emphasis, not just styling...
```suggestion
MAINTENANCE_MSG = {"text": "This site will be temporarily unavailable for maintenance <strong>{custom_message}</strong>.\nWe apologise for any inconvenience this may cause.", "level": "INFO", "type": "PLANNED_MAINTENANCE"}  # NOQA
```